{"name": "test_login_courier_missing_field_status_code[password]", "status": "failed", "statusDetails": {"message": "AssertionError: Ожидался код ответа 400, но получен 504\nassert 504 == 400\n +  where 504 = <Response [504]>.status_code", "trace": "self = <tests.test_login_courier.TestCourierLogin object at 0x101f3b7d0>, create_courier = ('cqockfsanl', 'avhyhywbwo', 'hpbfopcynv'), missing_field = 'password'\n\n    @pytest.mark.parametrize(\"missing_field\", [\"login\", \"password\"])\n    def test_login_courier_missing_field_status_code(self, create_courier, missing_field):\n        courier_data = create_courier\n        payload = {\n            \"login\": courier_data[0],\n            \"password\": courier_data[1]\n        }\n        del payload[missing_field]\n        response = requests.post(COURIER_LOGIN_URL, json=payload)  # Используем COURIER_LOGIN_URL из config\n>       assert response.status_code == 400, f\"Ожидался код ответа 400, но получен {response.status_code}\"\nE       AssertionError: Ожидался код ответа 400, но получен 504\nE       assert 504 == 400\nE        +  where 504 = <Response [504]>.status_code\n\ntests/test_login_courier.py:39: AssertionError"}, "parameters": [{"name": "missing_field", "value": "'password'"}], "start": 1728393834437, "stop": 1728393894753, "uuid": "44254795-e3a1-4ace-a62e-09ef2a14c781", "historyId": "e03cee4e3b2ce4e17da976409bcf40ad", "testCaseId": "23368a20f2c38ffdac9671f9bfc79a51", "fullName": "tests.test_login_courier.TestCourierLogin#test_login_courier_missing_field_status_code", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "MacBook-Pro-Daniil.local"}, {"name": "thread", "value": "56047-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}]}